### PR TITLE
Updated Active Record support docs

### LINKED ISSUE #7472: Update Active Record tutorial to use new adapter version
* The note [here](https://www.cockroachlabs.com/docs/v20.1/build-a-ruby-app-with-cockroachdb-activerecord.html) needs to be updated to say use `version 5.2.x of ActiveRecord requires version 5.2.x of the adapter`
* Mentions of beta support should be removed for Active Record.
* Version and support level needs to be updated on the [summary tooling page](https://www.cockroachlabs.com/docs/v20.1/third-party-database-tools.html)

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #7472.

### CODE REVIEW

**Comment by vy-ton** [comment]
@ericharmeling Just checked with Rafi, but the new adapter requires CRDB `v20.1` due to savepoint support. We should highlight this on the tooling summary page.

File: _includes/v20.1/misc/tooling.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If you encounter problems, please [open an issue](https://github.com/cockroachdb
 | Go | [GORM](build-a-go-app-with-cockroachdb-gorm.html) | [1.9.11](https://github.com/jinzhu/gorm/releases) | Full |
 | Java | [Hibernate](build-a-java-app-with-cockroachdb-hibernate.html)<hr>[jOOQ](build-a-java-app-with-cockroachdb-jooq.html) | [5.4](https://hibernate.org/orm/releases/)<hr>[3.13.2](https://www.jooq.org/download/versions) (must be 3.13.0 or higher) | Full<hr>Full |
 | Node.js | [Sequelize](build-a-nodejs-app-with-cockroachdb-sequelize.html) | [sequelize 5.21.9](https://www.npmjs.com/package/sequelize)<br>[sequelize-cockroachdb 1.1.0](https://www.npmjs.com/package/sequelize-cockroachdb) | Beta |
-| Ruby | [ActiveRecord](build-a-ruby-app-with-cockroachdb-activerecord.html) | [activerecord 5.2.4.2](https://rubygems.org/gems/activerecord)<br>[activerecord-cockroachdb-adpater 5.2.0.beta2](https://rubygems.org/gems/activerecord-cockroachdb-adapter) | Beta |
+| Ruby | [ActiveRecord](build-a-ruby-app-with-cockroachdb-activerecord.html) | [activerecord 5.2](https://rubygems.org/gems/activerecord)<br>[activerecord-cockroachdb-adpater 5.2.0](https://rubygems.org/gems/activerecord-cockroachdb-adapter) | Full |
```
Comment (vy-ton) [inline _includes/v20.1/misc/tooling.md:33]: Maybe we can say `Full as of  v20.1`?

**Comment by rafiss** [comment]
Perhaps we could say that there's beta support for 19.2 with the new adapter?

I believe it should work in general, it's just that we haven't run the full test suite against it because the tests themselves make heavy use of savepoints.

File: _includes/v20.1/misc/tooling.md (hunk 1 above)
Comment (ericharmeling) [reply _includes/v20.1/misc/tooling.md]: Done.

**Comment by ericharmeling** [comment]
@vy-ton 

I made the updates you requested. See the changes in the latest commit.

**Comment by vy-ton** [comment]
LGTM!

**Pull request merged**